### PR TITLE
Basic functionality of the portfolio website.

### PR DESCRIPTION
**Pull request by rebekadittrich**

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>YOUR_PROJECT_ID_HERErebekad-step-2020</deploy.projectId>
```
Comment (pmueller-gg) [inline portfolio/pom.xml]: remove YOUR_PROJECT_ID_HERE

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,33 @@
...
+          <a href="projects.html">Resume</a>
+          <a href="gallery.html">Gallery</a>
+        </div>
+      </div>
+      <div class="column">
+        <div id="content">
+          <h1>Hello.<br> I am Rebeka</h1>
+          <img src="/images/portfolio1.jpg" alt="Cat" style="width:500px;height:500px;">
+          <div class="text-content">
+            <p class="text-content">I'm a second year student at Imperial College London from Budapest, Hungary.
```
Comment (pmueller-gg) [inline portfolio/src/main/webapp/index.html:27]: > I'm a second year student at Imperial College London from Budapest, Hungary

The sentence seems grammattically ambiguous, I suggest to split it in two (optional).

**Review by pmueller-gg — APPROVED** [review]
LGTM, optional improvements inline.

File: portfolio/src/main/webapp/gallery.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          <h1>Gallery</h1>
+          <p>Few of my paintings and drawings.</p>
+          <img src="/images/gallery/g3.jpg">
+          <img src="/images/gallery/g2.JPG">
+          <img src="/images/gallery/g1.jpg">
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```
Comment (marcbury-g) [inline portfolio/src/main/webapp/gallery.html:33]: Please add a new line at the end of every file. That is just for being consistent that a line in a file always ends with a newline char (tooling maybe depends on this).

File: portfolio/src/main/webapp/projects.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+      </div>
+      <div class="column">
+        <div id="content">
+          <h1>Resume</h1>
+          <embed src= "cv.pdf" width= "595" height= "842">
+        </div>
+      </div>
+    </div>
+  </body>
+</html>
```
Comment (marcbury-g) [inline portfolio/src/main/webapp/projects.html:30]: Same here.

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,9 +1,71 @@
...
+  white 0px 0px 2px, white 0px 0px 1px, white 0px 0px 1px, white 0px 0px 1px, 
+  white 0px 0px 1px, white 0px 0px 1px, white 0px 0px 1px, white 0px 0px 1px, 
+  white 0px 0px 1px, white 0px 0px 1px, white 0px 0px 1px, white 0px 0px 1px;
+  -webkit-font-smoothing: antialiased;
+}
+
+.socialnav {
+  float: right;
+  padding: 30px;
+}
```
Comment (marcbury-g) [inline portfolio/src/main/webapp/style.css:71]: Same here.

**Review by marcbury-g — APPROVED** [review]
Just a few minor things.

**Pull request merged**